opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/README.adoc

```diff
@@ -242,6 +242,16 @@ org.opennms.web.rest.support.SearchPropertiesToAsciidocTest
 | event.ipAddr | IP Address | IP Address
 |===
 
+[[eventParameterProperties]]
+.Event Parameter Properties
+[options="header",width="99%",cols="2m,1,3"]
+|===
+| Name | Type | Description
+| eventParameter.name | String | IP Address
```

Comment:
IP Address?

Reply:
Ah, I fixed that in the code but forgot to re-run the doc generator. Good catch!